SquirrelInputController.m

```diff
@@ -166,10 +166,11 @@ -(BOOL)processKey:(int)rime_keycode modifiers:(int)rime_modifiers
                                          rime_keycode == XK_bracketleft));
     if (isVimBackInCommandMode) {
       NSString* app = [_currentClient bundleIdentifier];
-      if ([app isEqualToString:@"org.vim.MacVim"] &&
-          !rime_get_api()->get_option(_session, "ascii_mode")) {
+      SquirrelAppOptions *appOptions = [NSApp.squirrelAppDelegate.config getAppOptions:app.copy];
```

Comment:
Querying app options from config is expensive.
Instead, use `RimeApi::get_option()` to query the cached value in the IME session's context.